test/dest.js

```diff
@@ -763,53 +763,6 @@ describe('dest stream', function() {
     stream.end();
   });
 
-  it('should write file atime and mtime using the vinyl stat', function(done) {
```

Comment:
what do you mean by "But I am increasingly uncomfortable that we've got some of this stuff the wrong way around."?